enterprise/cmd/executor/internal/run/validate.go

```diff
@@ -71,6 +76,26 @@ func RunValidate(cliCtx *cli.Context, logger log.Logger, config *config.Config)
 	return nil
 }
 
+var AuthorizationFailedErr = errors.New("failed to authorize with frontend, is the authorization token set correctly?")
```

Comment:
```suggestion
var authorizationFailedErr = errors.New("failed to authorize with frontend, is the authorization token set correctly?")
```

Since it is not being used outside the package, it does not need to be exported